bin/tag-release

```diff
@@ -27,7 +27,9 @@ function cloneRepository(SymfonyStyle $ui, string $directory, string $branch, st
             $branch,
             $directory
         )
-    ))->run();
+    ))
+        ->setTimeout(3600)
```

Comment:
I think a comment explaining the timeout would be great

Reply:
@Prokyonn comment added